tests/testthat/error/test-subscript-loc.txt

```diff
@@ -501,5 +501,14 @@ vec_as_location() checks dimensionality
 =======================================
 
 > vec_as_location(matrix(TRUE, nrow = 1), 3L)
-Error: `i` must be a one-dimensional vector but has 2 dimensions.
+Error: Must subset elements with a valid subscript vector.
+x The subscript must be a simple vector, not a matrix.
```

Comment:
a 1d vector, not a matrix?

Reply:
This seems more technical than needed? I don't have a strong preference towards "simple" though.